### ISSUE TITLE
[10k]: Contribution process and document public guidelines

### ISSUE THREAD

**Issue by sumiat** [HHS/simpler-grants-gov#1230]

**Comment by margaretspring** [comment]
@btabaska @sumiat - Please check out this ticket & advise if it should be moved to the Code challenge work or if it's considered complete. Thank you!

@widal001 - All of this might be moot as this is a 10K & aren't they getting removed?

**Comment by sumiat** [comment]
@btabaska feel free to chime in here but I believe this 10k can be closed/marked as done as the work to complete this was done in this milestone: https://github.com/HHS/simpler-grants-gov/milestone/127?closed=1. This is just the 10k.

**Comment by margaretspring** [comment]
Yep - I just went & checked the milestones & saw it was just the 10K issue was left open - https://github.com/HHS/simpler-grants-gov/milestone/127

Thank you for checking @sumiat - I'll close now!
